### ISSUE TITLE
Can't re-install OBS Plugin from Marketplace

### ISSUE THREAD

**Issue by DJPVMA** [Loupedeck-open-source/Loupedeck-ObsStudio-OpenPlugin#40]
Loupedeck's OBS plugin stops switching scenes that I have programmed to the Loupedeck Device's buttons when I restart my Mac, so I uinstalled the OBS plugin. When I look at my Marketplace for OBS, the action button says "Uninstall" even though I already did that, and it won't let me re-install the OBS plugin. I deleted Loupedeck from my Mac and then reinstalled the Loupedeck app, but it still shows that OBS is already installed.

**Comment by Menrix** [comment]
have exactly the same issue.
Are there any solution?

**Comment by DJPVMA** [comment]
> have exactly the same issue. Are there any solution?

No, I have not found a solution. The guy who is in charge of support was useless and asked me the specific issue even though I told him exactly what he needed to know to understand the issue, and because of his incompetence and uselessness towards this issue that they can't keep just ignoring and hoping it goes away, I switched to Stream Dock. Loupedeck is now a $100 paperweight on my desk. Thanks, team Loupedeck!

**Comment by DJPVMA** [comment]
They should put you in charge of support. Thanks for sharing. I'll check into it.

**Comment by DJPVMA** [comment]
Thanks, it worked! Now, I can return this Stream Dock. 🙏🏻 The Stream Dock buttons won't stay programmed after a while.
